usr/src/lib/libzfs_core/common/libzfs_core.c

```diff
@@ -171,27 +171,32 @@ lzc_ioctl(zfs_ioc_t ioc, const char *name,
 }
 
 int
-lzc_create(const char *fsname, enum lzc_dataset_type type, nvlist_t *props)
+lzc_create(const char *fsname, enum lzc_dataset_type type, nvlist_t *props,
+    nvlist_t *hidden_args)
```

Comment:
Someone else commented about this in the mailing list and this is my response. Let me know if you agree or disagree:

> I could do that, if that is what the community wants (it would take
only a few minutes). However, the comment at the top of the file says:
"Currently, the interface is Evolving (not Committed), but we intend
to commit to it once it is more complete and we determine that it
meets the needs of all consumers." It seems to me that either the we
are committed enough to the interface that we should remove that note
or we should make changes to it now while it is still "evolving".